lxc_test.go

```diff
@@ -1051,6 +1051,36 @@ func TestIPAddress(t *testing.T) {
 	}
 }
 
+func TestIPv4Address(t *testing.T) {
+	c, err := NewContainer(ContainerName)
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+
+	if unprivileged() {
+		time.Sleep(3 * time.Second)
```

Comment:
Yeah, that wouldn't have been my choice either - I adapted these tests from existing the tests for related functionality already in the codebase :wink: